src/ext/php7/engine_hooks.c

```diff
@@ -752,7 +769,7 @@ static void _dd_return_helper(zend_execute_data *execute_data) {
             ZVAL_NULL(&rv);
             retval = &rv;
         }
-        _dd_end_span(span, retval);
+        ddtrace_posthook(NULL, span, retval);
```

Comment:
We should have access to the fbc from the `execute_data` here (and in the other return handlers). Or is there a reason we need to keep it `NULL`?

Reply:
It's because we don't need to check for a span stack mismatch because of the way the handlers works -- if there was a mismatch we wouldn't even be running the posthook in these handlers.

Reply:
Good point. Just FYI, that assumption will likely change in #931.